### PR TITLE
WIP: Remove Handles from Graph

### PR DESCRIPTION
**Pull request by joshuashort**

### CODE REVIEW

**Comment by hannahtuttle** [comment]
Just a Quick update: I have a netlify function that pulls in all the data from _data/handles  and returns an array of handles and I started replacing the call for judge-application.js . There are 4 places that I have seen that will need to be updated.

- [x] create netlify function for handles
- [ ] judge-application.js
- [ ] manage-team.tsx
- [ ] register-team.tsx
- [ ] register.tsx

**Comment by hannahtuttle** [comment]
> have you seen the existing `get-team` and `get-user` functions? Can those be used?

Yes, I did see that and I plan to use the `get-team` function and possibly the `get-user` functions as I need the team for at register.tsx but I haven't looked through register-team or manage-team yet but I imagine I will need it for those as well. I will be back on later tonight my time and should get another good chunk of this done then.

**Comment by joshuashort** [comment]
> > have you seen the existing `get-team` and `get-user` functions? Can those be used?
> 
> Yes, I did see that and I plan to use the `get-team` function and possibly the `get-user` functions as I need the team for at register.tsx but I haven't looked through register-team or manage-team yet but I imagine I will need it for those as well. I will be back on later tonight my time and should get another good chunk of this done then.

Nice -- thanks!

(and good find on the removable judge-application use!)

File: _data/handles/D-squared.json [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
```
Comment (joshuashort) [inline _data/handles/D-squared.json:1]: This file should stay?

**Comment by hannahtuttle** [comment]
@joshuashort I added that file in here, could you review these changes and then could we meet to talk about the next phase of these changes?

**Review by Simon-Busch — APPROVED** [review]

**Comment by natevw** [comment]
Adding @captainmangoC4 as an "assignee" here too (whatever that means exactly wrt to a PR?) just since it's possible they might be revisiting a lot of things related to handles during some upcoming function migration work.

**Comment by SamusElderg** [comment]
Closing stale PR

**Pull request closed without merging**